qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/java/AbstractQuteTemplateLinkCollector.java

```diff
@@ -138,7 +142,9 @@ public boolean visit(FieldDeclaration node) {
 							.getLocationExpressionFromConstructorParameter(variable.getName().getIdentifier());
 				}
 				String fieldName = variable.getName().getIdentifier();
-				collectTemplateLink(null, node, locationExpression, getTypeDeclaration(node), null, fieldName, false);
+				TemplateNameStrategy templateNameStrategy = TemplateNameStrategy.ELEMENT_NAME;

```

Comment:
this can be inlined directly in the collectTemplateLink call

Reply:
fixed